example/dags/snowflake_sample_dag.py

```diff
@@ -69,15 +69,23 @@
 SUPPORTED_SCHEMA_SQL_IN_CLAUSE = "('{schemas}')".format(schemas="', '".join(SUPPORTED_SCHEMAS))
 
 # SNOWFLAKE CONFIGs
-SNOWFLAKE_DATABASE_KEY = 'YOUR_SNOWFLAKE_DB_NAME'
+SNOWFLAKE_DATABASE = 'YOUR_SNOWFLAKE_DB_NAME'
 
 
 # todo: connection string needs to change
 def connection_string():
     user = 'username'
     password = 'password'
     account = 'YourSnowflakeAccountHere'
-    return "snowflake://%s:%s@%s" % (user, password, account)
+    # specify a warehouse to connect to.
+    warehouse = 'yourwarehouse'
+    return 'snowflake://{user}:{password}@{account}/{database}?warehouse={warehouse}'.format(
```

Comment:
I am not familiar with snowflake connection string format, you need to put the doc on how it defines the string here.

Reply:
Good point. I will add some comments and make it clear. Will update both the files.